site2/website-next/blog/2022-12-01-pulsar-summit-asia-2022-recap.md

```diff
@@ -0,0 +1,101 @@
+* **A Deep Dive into Pulsar's Geo-replication for High Availability (Mandarin)**: Jialing Wang from China Mobile talked about the asynchronous and synchronous data replication mechanisms, and explained how they deployed Pulsar across multiple regions and improved its performance at China Mobile.
+* **Apache Pulsar in Volcano Engine E-MapReduce: Integration and Scenarios (Mandarin)**: Xin Liang from ByteDance introduced Volcano Engine E-MapReduce, a stateless, open-source big data platform, and how Pulsar fits into the platform’s ecosystem supporting different use cases.
+* **Handling 100K Consumers with One Topic: Practices and Technical Details (English): **Hongjie Zhai from NTT Software Innovation Center shared their practices and technical details of handling 100K consumers with a single Pulsar topic.
+
+### Ecosystem
+
+* **Pulsar + Flink + Camel: How Vertice Built its CMDB-based Real-time Data Platform (Mandarin)**: Wei Wang from Vertice offered his insights on how to build a CMDB-based Real-time Data platform with Pulsar, Flink, and Camel.
+* **Simplify Pulsar Functions Development with SQL (Mandarin)**: Rui Fu from StreamNative discussed how SQL syntax, Pulsar Functions, and Function Mesh can work together to deliver a unique user development experience for real-time data jobs in the cloud environment.
+* **Apache Pulsar + KubeEdge: Managing Edge Devices with Low Latency and Persistent Storage (Mandarin)**: Ryan Zhao from Huawei Cloud introduced a management solution for edge devices implemented through the Device Management Interface of KubeEdge and Apache Pulsar.
+* **Make Querying from Pulsar Easier: Introduce Flink Pulsar SQL Connector (English)**: Yufei Zhang from StreamNative walked through the basic concepts and examples of using Pulsar SQL Connector and discussed PulsarCatalog’s two different modes of using Pulsar as a metadata store.
+* **Migrating from RabbitMQ to Apache Pulsar: Using AMQP-on-Pulsar (AoP) in E-commerce Industry (Mandarin)**: Yifei Ming from Access Corporate Group talked about their experience of using the AMQP-on-Pulsar project to migrate RabbitMQ workloads to AoP.
+
+## What’s new in the Pulsar community
+
+In addition to the keynotes and sessions, we also shared some exciting news at the Summit.
+
+Apache Pulsar has been adopted by organizations and users across the globe since it graduated as a Top Level Project in September 2018. Recently, the project witnessed its 580th contributor, almost hitting the 600 milestone!
+
+![](/img/pulsar-github-contributor-20221114.png)
+
+<small><center>Figure 1. Pulsar GitHub repo contributors</center></small>
+
+So far this year, we have welcomed 15 new Apache Pulsar [Committers](https://www.apache.org/foundation/how-it-works.html#committers) to the Pulsar family. They have made continuous contributions to the Pulsar community and as Pulsar Committers, they now have write access to the Pulsar repository. They are:
+
+* [@RobertIndie](https://github.com/RobertIndie)
```

Comment:
This list seems unsorted. I'd prefer to sort in alphabetical order or invited time order. But it's not a requirement :)